src/com/jwetherell/algorithms/mathematics/Division.java

```diff
@@ -3,11 +3,17 @@
 public class Division {
 
     public static final long division(int a, int b) {
+        if (b == 0) {
+            throw new IllegalArgumentException("Divisor can't be zero.");
+        }
         long result = ((long) a) / ((long) b);
         return result;
     }
 
     public static final long divisionUsingLoop(int a, int b) {
+        if (b == 0) {
```

Comment:
Added an if statement that checks if the b integer equals zero. If it does the program will throw an IllegalArgumentException and then display an error message.